src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/Commands/RemoveAzureContainerRegistryReplication.cs

```diff
@@ -0,0 +1,74 @@
+
+namespace Microsoft.Azure.Commands.ContainerRegistry
+{
+    [Cmdlet(VerbsCommon.Remove, ContainerRegistryReplicationNoun, DefaultParameterSetName = NameResourceGroupParameterSet, SupportsShouldProcess = true)]
+    [OutputType(typeof(bool))]
+    public class RemoveAzureContainerRegistryReplication : ContainerRegistryCmdletBase
+    {
+        [Parameter(Position = 0, Mandatory = true, ParameterSetName = NameResourceGroupParameterSet, HelpMessage = "Container Registry Replication Name. Default to the location name.")]
+        [ValidateNotNullOrEmpty]
+        [Alias(ReplicationNameAlias, ResourceNameAlias)]
+        public string Name { get; set; }
+
+        [Parameter(Position = 1, Mandatory = true, ParameterSetName = NameResourceGroupParameterSet, HelpMessage = "Resource Group Name.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Position = 2, Mandatory = true, ParameterSetName = NameResourceGroupParameterSet, HelpMessage = "Container Registry Name.")]
+        [Alias(ContainerRegistryNameAlias)]
+        [ValidateNotNullOrEmpty]
+        public string RegistryName { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = ReplicationObjectParameterSet, ValueFromPipeline = true, HelpMessage = "Container Registry Object.")]
+        [ValidateNotNull]
+        public PSContainerRegistryReplication Replicatoin { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = ResourceIdParameterSet, ValueFromPipelineByPropertyName = true, HelpMessage = "The container registry replication resource id")]
+        [ValidateNotNullOrEmpty]
+        [Alias(ResourceIdAlias)]
+        public string ResourceId { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            if (string.Equals(ParameterSetName, ReplicationObjectParameterSet))
+            {
+                ResourceId = Replicatoin.Id;
+            }
+            if (MyInvocation.BoundParameters.ContainsKey("ResourceId") || !string.IsNullOrWhiteSpace(ResourceId))
+            {
+                string resourceGroup, registryName, childResourceName;
+                if (!ConversionUtilities.TryParseRegistryRelatedResourceId(ResourceId, out resourceGroup, out registryName, out childResourceName)
+                    || string.IsNullOrEmpty(childResourceName))
+                {
+                    WriteInvalidResourceIdError(InvalidReplicationResourceIdErrorMessage);
+                    return;
+                }
+
+                ResourceGroupName = resourceGroup;
+                RegistryName = registryName;
+                Name = childResourceName;
+            }
+
+            if (ShouldProcess(Name, "Delete the replication from the container registry. This will also delete all the webhooks in this replication."))
+            {
+                RegistryClient.DeleteReplication(ResourceGroupName, RegistryName, Name);
+                WriteObject(true);
```

Comment:
@Jinhuafei the pattern for `Remove-*` cmdlets is to have a `PassThru` switch parameter, and if the user provides it, then you write the `bool` to the output stream, otherwise you write nothing. If you want to always write true, that is fine as well.